maps/dist/computer.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"computer.js","sourceRoot":"","sources":["../objects/computer.ts"],"names":[],"mappings":"AAAA,OAAO,KAAK,MAAM,MAAM,QAAQ,CAAC;AAEjC,IAAO,MAAM,GAAG,MAAM,CAAC,WAAW,CAAC,MAAM,CAAC;AAmC1C,MAAM,QAAS,SAAQ,MAAM;CAE5B;AAGD,eAAe;IACX,OAAO,EAAE,CAAC,MAAkC,EAAE,EAAE;QAC5C,MAAM,CAAC,KAAK,CAAC,UAAU,EAAE,wCAAwC,EAAE,+CAA+C,CAAC,CAAC;IACxH,CAAC;IACD,MAAM,EAAE,CAAC,KAAY,EAAE,EAAE;IAEzB,CAAC;IACD,OAAO,EAAE,CAAC,KAAY,EAAE,MAAuB,EAAE,EAAE;QAC/C,mHAAmH;QACnH,IAAI,GAAG,GAAG,IAAI,QAAQ,CAAC,KAAK,EAAE,MAAM,CAAC,CAAC,EAAE,MAAM,CAAC,CAAC,EAAE,UAAU,CAAC,CAAC;QAC9D,KAAK,CAAC,GAAG,CAAC,QAAQ,CAAC,GAAG,CAAC,CAAC;QACxB,mDAAmD;IACvD,CAAC;CACJ,CAAC"}
```

Comment:
This file should be removed and added to .gitignore

Reply:
same